src/components/Common/navbar.component.jsx

```diff
@@ -0,0 +1,56 @@
+import React, { Fragment } from 'react';
+import { Link } from 'react-router-dom';
+import Avatar from './avatar.component';
+
+const NavBar = ({ darkMode, setDarkMode }) => {
+  return (
+    <Fragment>
```

Comment:
This fragment isn't needed because the nav container already wraps all the content in this component.